common/php/util.php

```diff
@@ -40,20 +40,22 @@ function _ef_wp_link_page( $i, $custom_statuses ) {
 
 		if ( 1 == $i ) {
 			$url = get_permalink();
+		// phpcs:ignore Universal.ControlStructures.DisallowLonelyIf.Found
 		} else {
 			// Check for all custom post statuses, not just draft & pending
-			if ( '' == get_option('permalink_structure') || in_array($post->post_status, array_merge( $custom_statuses, array( 'pending' ) ) ) )
+			if ( '' == get_option( 'permalink_structure' ) || in_array( $post->post_status, array_merge( $custom_statuses, array( 'pending' ) ) ) ) {
 				$url = add_query_arg( 'page', $i, get_permalink() );
-			elseif ( 'page' == get_option('show_on_front') && get_option('page_on_front') == $post->ID )
-				$url = trailingslashit(get_permalink()) . user_trailingslashit("$wp_rewrite->pagination_base/" . $i, 'single_paged');
-			else
-				$url = trailingslashit(get_permalink()) . user_trailingslashit($i, 'single_paged');
+			} else if ( 'page' == get_option( 'show_on_front' ) && get_option( 'page_on_front' ) == $post->ID ) {
```

Comment:
Have we ever decided between `else if` vs `elseif`? Because I think there have been a PR or 2 before this where we've changed `else if` to `elseif`. 🤔

Reply:
Don't think so, though I'm not pedantic about that. I've always been used to writing `else if` hence why I stuck with it